src/com/lftechnology/java/training/niraj/ems/enums/Status.java

```diff
@@ -0,0 +1,25 @@
+package com.lftechnology.java.training.niraj.ems.enums;
+
+public enum Status {
+    ACTIVE("1"), INACTIVE("0");
+
+    private final String IS_ACTIVE = "Yes";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "IS_ACTIVE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)